### ISSUE TITLE
Fail to parse `assert` identifier in a call expression after import declaration without a semi-colon

### ISSUE THREAD

**Issue by KSXGitHub** [dprint/dprint-plugin-typescript#122]
Copy [`.dprintrc.json`](https://github.com/ksxnodemodules/static-type-assert/blob/46f31b60ce749d5e08eede4366e9b5dfec1743f9/.dprintrc.json) and [`test.ts`](https://github.com/ksxnodemodules/static-type-assert/blob/46f31b60ce749d5e08eede4366e9b5dfec1743f9/test.ts) from [commit 46f31b of ksxnodemodules/static-type-assert](https://github.com/ksxnodemodules/static-type-assert/tree/46f31b60ce749d5e08eede4366e9b5dfec1743f9) to reproduce the bug.

I know this is a `swc` bug, but I report it here since I don't know what version of `swc` that `dprint-plugin-typescript` uses.

---

**Describe the bug**

dprint-plugin-typescript version: 0.42.0

**Input Code**

```ts
import assert from './index'

assert<123>(123)
assert<123>(123, 123)
assert<123>(123, 123, 123)
```

**Expected Output**

run without errors

**Actual Output**

```
Error formatting test.ts. Message: Line 3, column 11: Expected a semicolon

  assert<123>(123)
            ~
Had 1 error(s) formatting.
```

**Comment by dsherret** [comment]
It is because of the identifier being called "assert".

Opened https://github.com/swc-project/swc/issues/1512

**Comment by dsherret** [comment]
Fixed in 0.43.0
